conf/config.level0.neon

```diff
@@ -32,8 +32,10 @@ rules:
 	- PHPStan\Rules\Functions\InnerFunctionRule
 	- PHPStan\Rules\Functions\NonExistentDefinedFunctionRule
 	- PHPStan\Rules\Functions\PrintfParametersRule
+	- PHPStan\Rules\Functions\TemplateTypeDeclarationRule
```

Comment:
I think it should rather be on level 2. Level 0 does not work with phpDocs at all.